src/main/java/net/datafaker/Nigeria.java

```diff
@@ -0,0 +1,37 @@
+package net.datafaker;
+
+import java.nio.file.Paths;
+import java.util.Locale;
+
+public class Nigeria {
+    private static final String KEY = "nigeria";
+    private final Faker faker;
+
+    protected Nigeria(Faker faker) {
+        this.faker = faker;
+        faker.fakeValuesService().addPath(Locale.ENGLISH, Paths.get("src/main/resources/en/nigeria.yml"));
```

Comment:
@bodiam @The-True-Hooha I know this PR has already been merged, but this location is still wrong. I guess we need to come up with a test that actually prevents building a JAR whenever this shows up, because the test will succeed but the actual JAR will not.

Reply:
Ah, thanks for letting me know, I even added an extra test to Faker It for this. No worries, I'll fix it.

Reply:
It's fixed, I removed this code, I already registered the yml file in the central spot.